.github/workflows/dev-stg-release.yml

```diff
@@ -7,63 +7,69 @@ on:
         description: Ballerina Central Environment
         type: choice
         options:
-        - STAGE
-        - DEV
+          - STAGE
+          - DEV
         required: true
 
 jobs:
-    release:
-        runs-on: ubuntu-latest
-        env: 
-          BALLERINA_${{ github.event.inputs.bal_central_environment }}_CENTRAL: true
-        steps:
-            - uses: actions/checkout@v2
-            - name: Set up JDK 11
-              uses: actions/setup-java@v1
-              with:
-                  java-version: 11
-            - name: Grant execute permission for gradlew
-              run: chmod +x gradlew
-            - name: Build with Gradle
-              env:
-                  packageUser: ${{ secrets.BALLERINA_BOT_USERNAME }}
-                  packagePAT: ${{ secrets.BALLERINA_BOT_TOKEN }}
-                  JAVA_OPTS: -DBALLERINA_DEV_COMPILE_BALLERINA_ORG=true
-              run: |
-                  ./gradlew build
+  release:
+    runs-on: ubuntu-latest
+    env:
+      BALLERINA_${{ github.event.inputs.bal_central_environment }}_CENTRAL: true
 
-            - name: Ballerina Build
-              uses: ballerina-platform/ballerina-action/@2201.7.0
-              with:
-                  args: pack ./twitter
-              env:
-                  JAVA_HOME: /usr/lib/jvm/default-jvm
+    steps:
+      - uses: actions/checkout@v2
 
-            - name: Run Trivy vulnerability scanner
-              uses: aquasecurity/trivy-action@master
-              with:
-                scan-type: 'rootfs'
-                scan-ref: '.'
-                format: 'table'
-                timeout: '10m0s'
-                exit-code: '1'
-              
-            - name: Push to Staging
-              if: github.event.inputs.bal_central_environment == 'STAGE'
-              uses: ballerina-platform/ballerina-action/@2201.7.0
-              with:
-                  args: push
-              env:
-                  WORKING_DIR: ./twitter
-                  JAVA_HOME: /usr/lib/jvm/default-jvm
-                  BALLERINA_CENTRAL_ACCESS_TOKEN: ${{ secrets.BALLERINA_CENTRAL_STAGE_ACCESS_TOKEN }}
-                  
-            - name: Push to Dev
-              if: github.event.inputs.bal_central_environment == 'DEV'
-              uses: ballerina-platform/ballerina-action/@2201.7.0
-              with:
-                  args: push
-              env:
-                  WORKING_DIR: ./twitter
-                  JAVA_HOME: /usr/lib/jvm/default-jvm
-                  BALLERINA_CENTRAL_ACCESS_TOKEN: ${{ secrets.BALLERINA_CENTRAL_DEV_ACCESS_TOKEN }}
+      # Set up Java Environment
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+
+      # Setup Ballerina Environment
+      - name: Set Up Ballerina
+        uses: ballerina-platform/setup-ballerina@v1.1.0
+        with:
+          version: latest
```

Comment:
For the `version` field, we should use the same version used in the old ballerina action (to avoid releasing with any unexpected version). This applies to `release` and `dev-tg-release` workflows in all the repositories. @nipunayf FYI